### PR TITLE
Add liberty profile

### PR DESCRIPTION
**Pull request by jonhawkes**

### CODE REVIEW

File: src/test/java/it/EndpointTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+public class EndpointTest {
+
+    public void testEndpoint(String endpoint, int expectedStatus, String expectedOutput) {
+        String port = System.getProperty("liberty.test.port");
```
Comment (kenfinnigan) [inline src/test/java/it/EndpointTest.java]: Maybe the property should be renamed to `test.port` and it defaults to 8080 but is overridden in Liberty Profile to another value if needed?

**Review by kenfinnigan — CHANGES_REQUESTED** [review]

File: src/test/java/it/EndpointTest.java (hunk 1 above)
Comment (jonhawkes) [reply src/test/java/it/EndpointTest.java]: yes. i'll do that.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+            <version>${version.download-maven-plugin}</version>
+            <executions>
+              <execution>
+                <id>install-tracer</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+                <configuration>
+                  <url>https://repo1.maven.org/maven2/net/wasdev/wlp/tracer/liberty-opentracing-zipkintracer/1.0/liberty-opentracing-zipkintracer-1.0-sample.zip</url>
```
Comment (Emily-Jiang) [inline pom.xml]: Do we still need this one as we can use service mesh zipkin installation?

File: src/main/profiles/liberty/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM open-liberty:webProfile7
+MAINTAINER IBM Java engineering at IBM Cloud
+COPY /target/liberty/wlp/usr/servers/defaultServer /config/
+COPY /target/liberty/wlp/usr/extension /opt/ol/wlp/usr/extension/
```
Comment (Emily-Jiang) [inline src/main/profiles/liberty/Dockerfile]: I don't think this is needed.

File: src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+    version="3.1">
+    <display-name>Microprofile Servicemesh Sample Service B</display-name>
```
Comment (Emily-Jiang) [inline src/main/webapp/WEB-INF/web.xml]: should say Service Mesh instead of Servicemesh

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (pilhuhn) [reply pom.xml]: a) I am not sure we can assume the Mesh has ZipKin. But yes, Jaeger can receive Zipkin data
b) The mesh can only provide you black-box tracing. If you want to know what happens inside the server/application, then you need to add MP-opentracing.

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thanks @jonhawkes !

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**